snc-library.sh

```diff
@@ -56,12 +57,17 @@ function run_preflight_checks() {
 
         echo "Checking libvirt and DNS configuration"
 
-        LIBVIRT_URI=qemu+tcp://localhost/system
+	LIBVIRT_URI=qemu:///system
+	if [ ${preset} == "snc" ]; then
+           LIBVIRT_URI=qemu+tcp://localhost/system
+	fi
 
         # check if libvirtd is listening on a TCP socket
-        if ! virsh -c ${LIBVIRT_URI} uri >/dev/null; then
```

Comment:
Imo it does not hurt to keep this even for microshift to check that we can access `qemu:///system`.